CONTRIBUTING.rst

```diff
@@ -0,0 +1,159 @@
+Contributing to Vulture
+=======================
+
+Creating and cloning a fork
+---------------------------
+
+Fork the repository on GitHub and do the following:
+
+::
+
+    $ git clone https://github.com/jendrikseipp/vulture.git
+    $ cd vulture
+    $ git remote add origin https://github.com/USERNAME/vulture
+
+Where ``USERNAME`` is your GitHub username.
+
+Installation
+------------
+
+Installing virtualenv
+~~~~~~~~~~~~~~~~~~~~~
+
+We recommend using ``virtualenv`` to isolate the installation of vulture.
+You can read more about
+it at the `virtualenv documentation <http://virtualenv.readthedocs.org>`_.
+
```

Comment:
We recommend using `virtualenv` to isolate the installation of vulture. 

(How to setup virtualenv?)

Refer to the [virtualenvdocs](http://virtualenv.readthedocs.org) for more info.